doc/z-chap6.xml

```diff
@@ -42,7 +45,7 @@ from} $E_a$ \emph{to} $E_b$. In this case we say that $E_a$ and $E_b$ are
     <#Include Label="DigraphEpimorphism">
     <#Include Label="EpimorphismsDigraphs">
     <#Include Label="GeneratorsOfEndomorphismMonoid">
-    <#Include Label="DigraphColoring">
+    <#Include Label="DigraphColouring">
```

Comment:
The synonym already exists in `grahom.gd`, and there are tests using both `DigraphColoring` and `DigraphColouring`. I just changed the name of the mansection's label.